src/typed-method-types/workflows/triggers/trigger-event-types.ts

```diff
@@ -0,0 +1,27 @@
+export const TriggerEventTypes = {
+  AppMention: "slack#/events/app_mentioned",
+  ChannelArchive: "slack#/events/channel_archived",
+  ChannelCreated: "slack#/events/channel_created",
+  ChannelDeleted: "slack#/events/channel_deleted",
+  ChannelRename: "slack#/events/channel_renamed",
+  ChannelShared: "slack#/events/channel_shared",
+  ChannelUnarchive: "slack#/events/channel_unarchived",
```

Comment:
We updated a bunch of these event strings, but didn't update the associated keys. I think we should unless there is a compelling argument not to